Cubical/Algebra/Group/Base.agda

```diff
@@ -1,8 +1,3 @@
-{-
```

Comment:
Why remove this header?

Reply:
I don't see the point in a file name Group.Base + "adds some smart constructors" is not prise and nor helpful to not what is inside

Reply:
It probably just refers to the make functions which are in every base file

Reply:
> These constructors are extra smart, so it would be better to document them in the header instead of removing it
Then what would like to add ? Just saying some smart constructors is not really useful.
Should we add something in all Base files that have such constructors ?

Reply:
Sounds good to me, I'll add that once the test is finished.